clcache.py

```diff
@@ -7,6 +7,7 @@
 # root directory of this project.
 #
 from ctypes import windll, wintypes
+import cProfile
```

Comment:
Do you know how much time this import takes? I think it was better placed in the CLCACHE_PROFILE block (plus ` # pylint: disable=wrong-import-position`)